.changeset/poor-weeks-act.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder': patch
+---
+
+Change the catagories filter to a dropdown component
```

Comment:
This should be `Categories`

Reply:
Thanks, I didn't see the typo. I just fix it